feathr_project/feathr/_databricks_submission.py

```diff
@@ -184,8 +184,10 @@ def wait_for_completion(self, timeout_seconds: Optional[int] = 600) -> bool:
                 # See here for the returned fields: https://docs.microsoft.com/en-us/azure/databricks/dev-tools/api/2.0/jobs#--response-structure-8
                 # print out logs and stack trace if the job is failed
                 logger.error("Feathr job is failed. Please visit this page to view error message: {}", self.job_url)
-                logger.error("Error Code: {}", result["error"])
-                logger.error("{}", result["error_trace"])
+                if "error" in result:
```

Comment:
if result["error"]?

Reply:
that won't work (it will throw out exception). `if "key" in dict` is recommended way to detect key in python dict